src/euphorie/client/model.py

```diff
@@ -398,15 +398,21 @@ class Account(BaseObject):
     password = schema.Column(types.Unicode(64))
     tc_approved = schema.Column(types.Integer())
     account_type = schema.Column(
-        Enum([u"guest", u"converted", None]),
-        default=None,
+        Enum([u"guest", u"converted", "full"]),
```

Comment:
This is missing an upgrade step, right?

Reply:
It should already be present, in the alembic upgrade: https://github.com/euphorie/Euphorie/pull/238/files#diff-fbef39df29989bc254ed99389d7fad78R21

Reply:
When you change an Enum (AFAIK) you have to update the type.
See https://stackoverflow.com/questions/1771543/adding-a-new-value-to-an-existing-enum-type